api/PerformanceNavigation.json

```
@@ -29,10 +29,10 @@
             "version_added": "14"
           },
           "safari": {
-            "version_added": false
+            "version_added": "11"
           },
           "safari_ios": {
-            "version_added": false
+            "version_added": "11"
```

```suggestion
            "version_added": "9"
```

This matches https://developer.mozilla.org/en-US/docs/Web/API/Performance/navigation#browser_compatibility and it looks like that data is correct, it shouldn't be iOS 8. Testing on iOS 8 and 9 on BrowserStack.

The same change can be made for `redirectCount` and `type`.